src/Symfony/Bridge/Twig/Extension/HttpFoundationExtension.php

```diff
@@ -12,7 +12,7 @@
 namespace Symfony\Bridge\Twig\Extension;
 
 use Symfony\Component\HttpFoundation\RequestStack;
-use Symfony\Component\HttpFoundation\Request;
```

Comment:
Good catch, PhpStorm doesn't seem to handle that properly. I changed the annotation to use the FQCN instead. Okay for you?